src/Catalyst.Abstractions/Kvm/IEthRpcService.cs

```diff
@@ -0,0 +1,53 @@
+using Catalyst.Abstractions.Kvm.Models;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Dirichlet.Numerics;
+
+namespace Catalyst.Abstractions.Kvm
+{
+    public interface IEthRpcService
+    {
+        //long eth_chainId();
+        //string eth_protocolVersion();
+        //SyncingResult eth_syncing();
+        //Address eth_coinbase();
+        //bool? eth_mining();
+        //ResultWrapper<byte[]> eth_snapshot();
+        //UInt256? eth_hashrate();
+        //UInt256? eth_gasPrice();
+        //IEnumerable<Address> eth_accounts();
+        long? eth_blockNumber();
+        UInt256? eth_getBalance(Address address, BlockParameter blockParameter);
+        ResultWrapper<byte[]> eth_getStorageAt(Address address, UInt256 positionIndex, BlockParameter blockParameter);
+        UInt256? eth_getTransactionCount(Address address, BlockParameter blockParameter);
+        //UInt256? eth_getBlockTransactionCountByHash(Keccak blockHash);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4411902)